packages/sdk/chainDeploy/helpers/oracles/uniswap.ts

```diff
@@ -71,6 +72,12 @@ export const deployUniswapOracle = async ({
         tokenPair.baseToken
       );
       console.log(oldBaseTokenOracle, "oldBaseTokenOracle updated?");
+
+      const uniswapV2Factory = new ethers.Contract(deployConfig.uniswap.uniswapV2FactoryAddress, [
```

Comment:
another way to use the uniswap factory to get the pair https://github.com/Midas-Protocol/monorepo/blob/ff0f6bb2061a05fde2c8c635269a8ab978622861/packages/sdk/src/modules/liquidation/getPotentialLiquidation.ts#L127